x-pack/plugins/cases/server/features.ts

```diff
@@ -28,9 +28,20 @@ export const getCasesKibanaFeature = (): KibanaFeatureConfig => {
   const apiTags = getApiTags(APP_ID);
 
   return {
+    deprecated: {
+      // TODO: Add docLinks to link to documentation about the deprecation
```

Comment:
Just curious, do we need to document the deprecation? @azasypkin I thought the deprecated roles had not surfaced to the users in the UI. Is that correct?

Reply:
You’re correct that we don’t currently surface roles with deprecated privileges in the UI, but we eventually will, see https://docs.google.com/document/d/1ajynT5qULXGlXaiHCPAKDCHNzCD2LwMHdL9M-XyEb6s/edit#heading=h.62r2judjxavq.

Generally, it’s a good idea to document any significant changes in our RBAC so that we can reference them later in release notes, SDHs, and any deprecation messages when we need to explain to users why they need to take action (e.g., manually migrate existing roles to use new privileges).

Reply:
We'll need to sync with docs then on how to document this properly